src/SDKs/Azure.Base/data-plane/Azure.Base/UriExtensions.cs

```diff
@@ -2,22 +2,207 @@
 // Licensed under the MIT License.
 
 using System;
+using System.Text;
```

Comment:
nit:  You may want to rename the file to reflect the new class name.

Reply:
Good point